mlflow/ml-package-versions.yml

```diff
@@ -13,7 +13,7 @@ sklearn:
 
   autologging:
     minimum: "0.20.3"
-    maximum: "0.24.1"
+    maximum: "0.24.2"
```

Comment:
A dummy change to make sure the proposed change works fine.

Reply:
It worked:

![image](https://user-images.githubusercontent.com/17039389/116517333-c5026380-a909-11eb-94d9-5210707fa49c.png)